### ISSUE TITLE
Bible OL terms glossary

### ISSUE THREAD

**Issue by jloewenstein** [EzerIT/BibleOL#61]
It's amazing how deep Bible OL can go. Sometimes too deep for me. 
With a limited background of linguistic terms, when I look through 'MyView' I can understand somethings immediately (i.e.: Person, gender, number), Somethings I can understand when I see examples such as (i.e.: State: Absolute/Construct). but some things are riddles I can't figure out, either because I don't understand even when looking at the data (i.e.: Syntactic code), or some that show 'None' or '-' for an entire passage so I don't even have data to try to understand (i.e.: Univalent final). I saw the help page, but couldn't find information there. 
Is there a lexicon for these terms? 
I'm especially interested as instructor, to be able to understand by what parameters can I quarry.

**Comment by tmccormack165** [comment]
Hi @jloewenstein, another excellent suggestion, we could even implement a "hover over" feature that gives definitions for terms when you hover over.

**Comment by oliverglanz** [comment]
@tmccormack165 that is a great idea (hover over feature) and I would be willing to help on writing content for the hover popup.

**Comment by ernstboogert** [comment]
You are right @jloewenstein that more work needs to be done on the proper explanation of grammar terms in the ETCBC database. We will look into it to find a nice solution. The tiptools are excellent, but before we will need a proper sentence explaining each term on its own. I remember that we had a student who worked on it in the past. I will look it up....
